src/main/scala/uk/gov/hmrc/play/http/HttpGet.scala

```diff
@@ -34,4 +34,19 @@ trait HttpGet extends HttpVerb with ConnectionTracing with HttpHooks {
     executeHooks(url, GET_VERB, None, httpResponse)
     mapErrors(GET_VERB, url, httpResponse).map(response => rds.read(GET_VERB, url, response))
   }
+
+  def GET[A](url: String, queryParams: Seq[(String, String)])(implicit rds: HttpReads[A], hc: HeaderCarrier): Future[A] =withTracing(GET_VERB, url) {
```

Comment:
Your new function is called wrapped in `withTracing` and the call to GET is also called `withTracing` so in the logs it will appear as it 2 http calls have been made when your new function is called.